recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "4.1.1" %}
+{% set version = "5.1.2" %}
 
 package:
   name: ipywidgets
   version: {{ version }}
 
 source:
   fn: ipywidgets-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/i/ipywidgets/ipywidgets-{{ version }}.tar.gz
-  md5: 491affdef8e0586eeeb299226ff634aa
+  url: https://github.com/ipython/ipywidgets/archive/{{ version }}.tar.gz
+  sha256: bdd8a4e6736e1240ce20a975f838c1f9a5cd044df733657d354af53267ab21df
 
 build:
   number: 0
-  script: python setup.py install
+  script: python -m pip install --no-deps .
```

Comment:
Thanks @minrk!

Pinging @msarahan and @jakirkham due to our recent discussion. I don't have a strong preference, but I would like to stick 1 (maybe 2) ways to call `script`.